pyo3-derive-backend/src/pymethod.rs

```diff
@@ -182,7 +182,7 @@ fn impl_wrap_init(cls: &syn::Type, name: &syn::Ident, spec: &FnSpec<'_>) -> Toke
         unsafe extern "C" fn __wrap(
             _slf: *mut ::pyo3::ffi::PyObject,
             _args: *mut ::pyo3::ffi::PyObject,
-            _kwargs: *mut ::pyo3::ffi::PyObject) -> libc::c_int
+            _kwargs: *mut ::pyo3::ffi::PyObject) -> ::std::os::raw::c_int
```

Comment:
Is this required? I would really like to avoid having to stick with the 2015 edition for the derive only for the doctest. (We should be able pass `--extern=2018` to rustdoc to avoid this if it doesn't work otherwise)

Reply:
Even when passing the edition directly to rustdoc, I get
```
---- guide/src/class.md - Python_Class::Object_properties (line 232) stdout ----
error[E0433]: failed to resolve: use of undeclared type or module `libc`
  --> guide/src/class.md:240:1
   |
10 | #[pymethods]
   | ^^^^^^^^^^^^ use of undeclared type or module `libc`

thread 'guide/src/class.md - Python_Class::Object_properties (line 232)' panicked at 'couldn't compile the test', src/librustdoc/test.rs:354:13
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

---- guide/src/class.md - Python_Class::Object_properties (line 261) stdout ----
error[E0433]: failed to resolve: use of undeclared type or module `libc`
  --> guide/src/class.md:269:1
   |
10 | #[pymethods]
   | ^^^^^^^^^^^^ use of undeclared type or module `libc`

thread 'guide/src/class.md - Python_Class::Object_properties (line 261)' panicked at 'couldn't compile the test', src/librustdoc/test.rs:354:13

---- guide/src/class.md - Python_Class::Object_properties (line 289) stdout ----
error[E0433]: failed to resolve: use of undeclared type or module `libc`
 --> guide/src/class.md:292:1
  |
5 | #[pyclass]
  | ^^^^^^^^^^ use of undeclared type or module `libc`

thread 'guide/src/class.md - Python_Class::Object_properties (line 289)' panicked at 'couldn't compile the test', src/librustdoc/test.rs:354:13
```